Completion/Unix/Command/_stow

```diff
@@ -0,0 +1,89 @@
+#compdef stow chkstow
+
+#
+# A zsh completion script for GNU stow (https://www.gnu.org/software/stow/)
+#
+
+__stow_complete_packages() {
+  local stow_dir=${(Q)1}
+  local -a stow_pkg_list=( $stow_dir/*(-/N:t) )
+
+  if [[ ${#stow_pkg_list} -gt 0 ]]; then
+    _values -C "packages from $stow_dir" ${stow_pkg_list[@]}
```

Comment:
You need to escape special characters in the operands if you're going to use `_values`; something like: `${${stow_pkg_list//\\/\\\\}//:/\\:}`

Alternatively you could replace all of this by `_wanted` and `compadd`, but you might have to change the way you call the function from `_arguments`

Also, descriptions should be singular (*package*, not *packages*)

Reply:
done in 3c75ffcc4